collection/network/capture-network-configuration-via-ipconfig.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: capture network configuration via ipconfig
+    namespace: collection/network
+    author: "@_re_fox"
+    scope: basic block
+    examples:
+      - 7204e3efc2434012e13ca939db0d0b02:0x403028
+  features:
+    - and:
+      - string: /ipconfig(\.exe)?/i
+      - api: msvcr100.system
```

Comment:
i agree that here is a place that we can expand.

one way of doing this might be to introduce a new rule "run shell command" (or something) and then reference it from here. lets do that when we have more examples.